bedrock/firefox/views.py

```diff
@@ -337,6 +337,13 @@ class MobileRelease:
         context.update(
             download_url=download_url,
         )
+        if product_slug == "desktop-esr":
+            download_esr_115_url = list(filter(lambda b: b["locale"] == locale, firefox_desktop.get_filtered_full_builds("esr115")))[0]["platforms"][
+                platform
+            ]["download_url"]
+            context.update(
+                download_esr_115_url=download_esr_115_url,
+            )
```

Comment:
There's a chance in the future that if/when `esr115` goes away this would throw an `IndexError`, similar to below for `esr_next`. We should probably catch that and wrap the added template code in a conditional if the esr 115 is there.

Reply:
Good suggestion, thanks!